### PR TITLE
docs: migrate from nektos/act README & wiki

### PR DESCRIPTION
**Pull request by panekj**

### CODE REVIEW

File: src/usage/custom_engine.md
Comment (ChristianSch) [inline src/usage/custom_engine.md]: should be "use a different container engine" probably

File: src/not_supported.md
Comment (ChristopherHX) [inline src/not_supported.md]: This is a gitea issue, not act issue

File: src/not_supported.md
Comment (ChristopherHX) [inline src/not_supported.md]: This is a gitea issue, not act issue

**Comment by cplee** [comment]
@catthehacker - ready for review?

File: src/installation/index.md
Comment (ChristopherHX) [inline src/installation/index.md]: Minor
```suggestion
- [GitHub CLI](./gh.md) (`Linux`, `macOS`, `Windows`, `FreeBSD`)
```

See assets of gh-act https://github.com/nektos/gh-act/releases/tag/v0.2.43, needs `-self-hosted` switch.

File: src/SUMMARY.md
Comment (ChristopherHX) [inline src/SUMMARY.md:19]: This is greyed out, is this expected?

image

File: src/beginner/index.md
Comment (ChristopherHX) [inline src/beginner/index.md]:
```suggestion
### via input or input-file flag
```
Subsection of "Pass Inputs to Manually Triggered Workflows"

File: src/beginner/index.md
Comment (ChristopherHX) [inline src/beginner/index.md]:
```suggestion
### via JSON
```

Subsection of "Pass Inputs to Manually Triggered Workflows"

File: src/SUMMARY.md
Comment (ChristopherHX) [inline src/SUMMARY.md:21]:
```suggestion
  - [Docker context](./missing_functionality/docker_context.md)
- [Known issues](./known_issues/index.md)
```

`missing_functionality/docker_context.md` has not been compiled for me, because it was not in summary

File: src/integrations.md
Comment (ChristopherHX) [inline src/integrations.md:5]:
```suggestion
- [Gitea Actions runner](https://gitea.com/gitea/act_runner)
- [github-act-runner](https://github.com/ChristopherHX/github-act-runner)
```

I also made a self-hosted runner with `nektos/act`, but not with the `Gitea Actions protocol` instead using the `Github Actions protocol`.

**Review by ChristopherHX — COMMENTED** [review]
The mdbook looks pretty nice and I don't need to learn a new docu language.

I wonder how the access rights are about this repo. Could the merge button be pressed if 1/1 approvals are given?.

File: src/SUMMARY.md
Comment (panekj) [reply src/SUMMARY.md:19]: Yes

Comment (ChristopherHX) [reply src/SUMMARY.md:19]: Good to know :)

**Review by ChristopherHX — APPROVED** [review]
Nice migration

**Comment by ChristopherHX** [comment]
We can merge this now if you want, seems like maintainer access level is enough after 1 non author review.

**Comment by cplee** [comment]
Go for it @ChristopherHX

**Pull request merged**
